pkg/cmd/version/version.go

```diff
@@ -79,6 +101,8 @@ func Command(p cli.Params) *cobra.Command {
 		},
 	}
 
+	cmd.Flags().StringVarP(&component, "component", "C", "", "provide a paticular component name for it's version (client|pipeline|triggers)")
```

Comment:
I would suggest to not have the `C` shorthand. Also please edit the description to something like below:

```suggestion
	cmd.Flags().StringVar(&component, "component",  "", "provide a component name (pipeline|triggers) to get its version")
```